test/test_utils/__init__.py

```diff
@@ -265,7 +265,12 @@ def request_tensorflow_inference(model_name, ip_address="127.0.0.1", port="8501"
     :connection: ec2_connection object to run the commands remotely over ssh
     :return:
     """
-    inference_string = "'{\"instances\": [1.0, 2.0, 5.0]}'"
+    
+    if "mnist_neuron" in model_name:
+        inference_string = "'{\"instances\": " + "{}".format([0 for i in range(784)]) + "}'"
```

Comment:
Fix neuron request for TF inference test.

Reply:
This is to fix single node neuron test. The bucket exist in us-east-1 region. Reverting the change as it will be covered in #1376 PR.